src/test/java/com/wikia/webdriver/testcases/messagewall/MessageWallTests.java

```diff
@@ -1,64 +1,83 @@
 package com.wikia.webdriver.testcases.messagewall;
 
+import com.wikia.webdriver.common.core.TestContext;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.TestContext'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)